bin/crew

```diff
@@ -1479,14 +1485,18 @@ def build_package(pwd)
 end
 
 def archive_package(pwd)
-  unless CREW_NOT_USE_PIXZ || !File.exist?("#{CREW_PREFIX}/bin/pixz")
-    puts "Using pixz to compress archive."
-    pkg_name = "#{@pkg.name}-#{@pkg.version}-chromeos-#{@device[:architecture]}.tpxz"
+  unless %x[tar --version | head -n 1 | awk '{print $4}'] < '1.31' || !File.exist?("#{CREW_PREFIX}/bin/zstd") || @pkg.no_zstd?
```

Comment:
Would it make sense to convert this into a crew constant?  Something like `CREW_TAR_VERSION`?

Reply:
We only need to know that when we are using tar. Why do that check every single time crew is run?

Reply:
Getting that value will make crew 0.010s to 0.157s slower/run, comparing i686 and x86_64 runs of that command.

Reply:
Fair enough.  I generally like to abstract kludgy logic like this, however.